spec/tags/flow-report.md

```diff
@@ -0,0 +1 @@
+Report to show datas about sessions in a period, it can be filtered by flow, dispatch and session.
```

Comment:
"Data" não tem plural em inglês. E talvez em vez de "data" usaria "state". Algo do tipo:
Report to provide current state of sessions (executions) of flows in a period. These states contain session variables and their current value. The report can be filtered by flow, dispatch or session.